File: 단계별로_풀어보기/2. 조건문/2. 9498.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString()
const score = parseInt(inputs)

if (score >= 90) {
  process.stdout.write('A')
} // ::
else if (score >= 80) {
  process.stdout.write('B')
} // ::
else if (score >= 70) {
  process.stdout.write('C')
} // ::
else if (score >= 60) {
  process.stdout.write('D')
} // ::
else {
  process.stdout.write('F')
}

File: 1. 단계별로_풀어보기/2. 조건문/2. 9498.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

score = int(input())

scoreDiv10 = score // 10

if scoreDiv10 >= 9:
  output(f"A")
elif scoreDiv10 >= 8:
  output(f"B")
elif scoreDiv10 >= 7:
  output(f"C")
elif scoreDiv10 >= 6:
  output(f"D")
else:
  output(f"F")